BUG(circular-loader) - fix

cedi315 opened the pull request:
made circular position: absolute
adjusted padding from top for headline Rooms

cedi315 committed:
BUG(circular-loader) - fix

made circular position: absolute
adjusted padding from top for headline Rooms

chrisWolter reviewed (changes requested):
![image](https://user-images.githubusercontent.com/48556854/100319542-13f54700-2fc0-11eb-8b70-9b771f195e0e.png)
Wenn wir hier die den Gebäude- oder Standortnamen angeben, dann wird die Schrift vom Circular Loader überdeckt.

cedi315 committed:
Merge branch 'master' into BUG-circular-loader

cedi315 committed:
Update Card.vue

adjusted position of circular loader to ensure there is no overlapping when card title is longer
